### ISSUE TITLE
Chimera.uchime - filename length error

### ISSUE THREAD

**Issue by mothur-westcott** [mothur/mothur#226]
"To add information, the old path before renaming (threw the error) was 211 characters (including separators); the new path after renaming (ran fine) was 166 characters."

http://www.mothur.org/forum/viewtopic.php?f=3&t=1560&start=20

**Comment by campenr** [comment]
Hello

So I've done a little more research to try and narrow this down. I used debug mode, and reran the command using the long file name (shortened by two characters) that was giving me the error. The first few groups ran through uchime fine, while the third failed. Looking at the actual commands passed to uchime the only difference was the --uchimeout option, whereby strings of 93 characters or less appear to work fine (full filepath = 257 characters), while strings 94 characters and greater did not (full filepath = 258 characters). The output of this command is available [here](http://pastebin.com/Kg5eHtN1) (whith the debug output for reading in the data removed for brevity).

I also reran the command identically (same location, same files) to how I ran it the first time I had this error (as reported on the mothur forum) with the logfile available [here](http://pastebin.com/G5Uk5f18). In this original (failing) command the option for --uchimeout was 95 characters (full filepath = 260 characters).

I also mentioned however that I had previously run this command successfully in a different directory (with a shorter path in terms of characters) without having to shorten the file name (full filepath = 210 characters).

If I execute these commands directly through uchime instead of through mothur I get identical results.

In summary, uchime appears not to be able to correctly access files with full paths of greater than 257 characters.

**Comment by mothur-westcott** [comment]
Completed with 8c16ba61b92564b657aa3026eb142d30e83f0009.
